### ISSUE TITLE
No gifs

### ISSUE THREAD

**Issue by sitolam** [yuepem/Exercises_Library#1]
I installed it but there are no images or gifs.

**Comment by yuepem** [comment]
cause the gifs-URL are all expired.

**Comment by sitolam** [comment]
> cause the gifs-URL are all expired.

Ow that's bad. Is there any way to access them?

**Comment by yuepem** [comment]
Yes, you can look into Rapid API and search ExerciseDB. This project was a practice, so I just used hardcoded for testing.  If you want to know more about this project, this is my email address: yuepem@gmail.com.
